tests/test_metrics.py

```diff
@@ -396,6 +396,41 @@ def test_inventory_quantity_per_gwe():
     obs = metrics.inventory_quantity_per_gwe.func(inv, tsp)
     assert_frame_equal(exp, obs)
 
+	
+def test_transaction_quantity_per_gwe():
+    #exp is the expected output metrics
+    exp = pd.DataFrame(np.array([
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 1, 7, 3, 3,  10, 20, 'LWR Fuel', 'kg/GWe', 0.82),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 2, 8, 4, 3,  20, 30, 'FR Fuel', 'kg/GWe', 0.61),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 3, 9, 5, 12, 30, 40, 'Spent Fuel', 'kg/GWe', 0.09),
```

Comment:
lines over 80 chars

Reply:
Wouldn't it be confusing if I separate this line?

Reply:
no, PEP8 recommandation are line should be shorter than 80 chars to improve readability,

I might seem unintuitive, but this way readers don't have to scroll horizontally